api-server/scanner-lib/src/sync/tests/simulation.rs

```diff
@@ -113,6 +121,34 @@ impl<'a> PoSAccountingView for PoSAccountingAdapterToCheckFees<'a> {
     }
 }
 
+struct StubTokensAccounting;
```

Comment:
I thought we used the word "Mock" instead of "Stub"... it's a matter of convention.

Reply:
Idk it's actually a stub with hardcoded state to satisfy function signature. It doesn't have any logic or expectation as mocks